Client/Frontend/ClientPreferences.swift

```diff
@@ -125,6 +125,8 @@ extension Preferences {
     
     /// Global Page Zoom-Level
     static var defaultPageZoomLevel = Option<Double>(key: "general.default-page-zoom-level", default: 1.0)
+    
+    static let isUsingBottomBar = Option<Bool>(key: "general.bottom-bar", default: false)
```

Comment:
What do you think about setting this default value to true? 
And migrate existing users to `false` at app launch.

Now it's not clear at first glance that new users have the bottom bar set as their default

Reply:
Im not sure if we can do that? We only know if it's their first launch, not first launch since adding a pref. We'd have to store a second property to check if we've already made the launch check, no?

Reply:
True, i can't think of a good way to do it. Let's keep it this way